src/components/ArticleView.jsx

```diff
@@ -36,23 +37,29 @@ const ArticleView = () => {
 	);
 
 	return (
-		<Card>
-			<Card.Body>
-				<Card.Img variant="top" src={article.article_img_url} />
-				<Card.Body></Card.Body>
-				<Card.Title>{article.title} </Card.Title>
-				<p>
-					by {article.author} | {dateDisplay} | 💬{' '}
-					{article.comment_count}
-				</p>
-				<Button variant="primary">👍</Button>
-				<Button disabled={true} variant="secondary">
-					{article.votes}
-				</Button>
-				<p></p>
-				<Card.Text align="left">{article.body} </Card.Text>
-			</Card.Body>
-		</Card>
+		<>
+			<Card>
+				<Card.Body>
+					<Card.Img variant="top" src={article.article_img_url} />
+					<Card.Body></Card.Body>
+					<Card.Title>{article.title} </Card.Title>
+					<p>
+						by {article.author} | {dateDisplay} | 💬{' '}
+						{article.comment_count}
+					</p>
+					<Col>
+						<Button variant="primary">👍</Button>
+						<Button variant="warning">👎🏽</Button>
+						<Button disabled={true} variant="secondary">
+							{article.votes}
+						</Button>
+					</Col>
+					<p></p>
+					<Card.Text align="left">{article.body} </Card.Text>
+				</Card.Body>
+			</Card>
+			<Commments id={id}></Commments>
+		</>
```

Comment:
You've done a great job with styling, layout and using bootstrap here Abdul, nice one!
It might be good to add the date the comment was created to the comment - but not mandatory